elated-egrets/bangazon-api
# Pzc product

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git pull pzc_product
git checkout pzc_product
python manage.py runserver
```

## Steps to test
Make sure you're in a virtual environment
Nuke migrations and database
In Bangazon project folder:
`python manage.py makemigrations Api`
`python manage.py migrage`
`python manage.py runserver`

Navigate to local/api/
See product endpoint is available
Add a user and category ID
Add a product to test adding a product with user and category ID


## What you changed

Added product_model.py, product_serializer.py, product_view.py
Edited all __init__.py files and urls.py

## What was added to read me
Nothing
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
1. Has the readme been updated to reflect changes made?
1. Has a teammate approved the PR?

Then Ship It!
